scanner/src/main/kotlin/experimental/ExperimentalScanner.kt

```diff
@@ -326,6 +326,16 @@ class ExperimentalScanner(
         return nestedProvenanceScanResults
     }
 
+    private fun Collection<Package>.filterNotConcluded(): Collection<Package> =
+        takeUnless { scannerConfig.skipConcluded }
+            ?: partition { it.concludedLicense != null && it.authors.isNotEmpty() }.let { (skip, keep) ->
```

Comment:
Why did you decide to check for authors?
To me this functionality seems unexpected from the name of the configuration option.

Reply:
This matches the current implementation in

https://github.com/oss-review-toolkit/ort/blob/8f4e9c18f5fa9f54dd076866351b0c084a411fcb/scanner/src/main/kotlin/Scanner.kt#L47-L56

until we have

https://github.com/oss-review-toolkit/ort/issues/4519

Reply:
@mnonnenmacher how about adding to the commit message that "authors" is only checked for consistency with the existing impl?

Reply:
That wasn't my decision, it's how the option was designed, see:

https://github.com/oss-review-toolkit/ort/blob/0ed3b248ae30f17bdebf330b71fe6357b33b4c12/model/src/main/kotlin/config/ScannerConfiguration.kt#L37-L41